src/lib.rs

```diff
@@ -79,7 +79,8 @@ use std::ops::{Deref,DerefMut};
 #[cfg(all(unix, not(target_os = "macos")))]
 extern crate dbus;
 #[cfg(all(unix, not(target_os = "macos")))]
-use dbus::{Connection, ConnectionItem, BusType, Message, MessageItem, Error};
+use dbus::{Connection, ConnectionItem, BusType, Message, MessageItem};
+pub use dbus::Error;
```

Comment:
woops, that commit should not be in this PR, but it is necessary if you want to return the Error.